m0t1fs/linux_kernel/file.c

```diff
@@ -3091,13 +3091,17 @@ static int pargrp_iomap_dgmode_recover(struct pargrp_iomap *map)
 		rc = -EIO;
 		goto end;
 	}
+
+#if 0 /* NA_FOR_INTEL_ISA */
 	if (parity_math(map->pi_ioreq)->pmi_parity_algo ==
 	    M0_PARITY_CAL_ALGO_REED_SOLOMON) {
 		rc = m0_parity_recov_mat_gen(parity_math(map->pi_ioreq),
 				(uint8_t *)failed.b_addr);
 		if (rc != 0)
 			goto end;
 	}
+#endif /* NA_FOR_INTEL_ISA */
+
```

Comment:
The whole part of code will be removed later as step 2 or step 3?

Reply:
We will not remove this code. We can keep it in #if 0 for future reference. Only temporary code added to support kernel space compilation will be removed later.

Reply:
OK.